daemon/api_debug.go

```diff
@@ -136,22 +189,7 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 	case "connectivity":
 		return checkConnectivity(st)
 	case "change-timings":
-		chg := st.Change(a.Params.ChgID)
-		if chg == nil {
-			return BadRequest("cannot find change: %v", a.Message)
-		}
-		m := map[string]struct {
-			DoingTime, UndoingTime time.Duration
-		}{}
-		for _, t := range chg.Tasks() {
-			m[t.ID()] = struct {
-				DoingTime, UndoingTime time.Duration
-			}{
-				DoingTime:   t.DoingTime(),
-				UndoingTime: t.UndoingTime(),
-			}
-		}
-		return SyncResponse(m, nil)
+		return getChangeTimings(st, a.Params.ChgID)
```

Comment:
can we change these to use `getDebug` (i.e. be a GET instead of a POST)?

Reply:
Yes, but I think it's a material for a small followup (happy to do it), current DebugGet implementation assumes just one "aspect" argument, I'd prefer not to mix it with this PR.

Reply:
I've addressed this in #6678